### PR TITLE
test: place path card test 5

### PR DESCRIPTION
**Pull request by nana-yang-hy**
complete test: place path card test 5

### CODE REVIEW

File: apps/backend/src/board/logic/place-path-card.test.ts [hunk 1]
```diff
@@ -266,5 +266,34 @@ describe("place path card", () => {
       - event source should includes
         - path card has been placed event with
           - path card [connected cross] at position (0, 1)
-  `);
+  `, async () =>
+    placePathCard(
```
Comment (kayac-chang) [inline apps/backend/src/board/logic/place-path-card.test.ts:270]: @nana-yang-hy 
In this scenario, by reference 
```
    given:
      a board with
        - a start card at position (0, 0)
```
we want to **place the a start card at position (0, 0)** first, 
先放 一張 start card 在 position (0, 0)

after that, then place **path card [connected cross] at position (0, 1)**
接著，再放 path card [connected cross] 在 position (0, 1)

**Review by kayac-chang — CHANGES_REQUESTED** [review]

File: apps/backend/src/board/logic/place-path-card.test.ts (hunk 1 above)
Comment (nana-yang-hy) [reply apps/backend/src/board/logic/place-path-card.test.ts:270]: @kayac-chang 
再次修改好test內容，再麻煩確認，感謝～

Comment (kayac-chang) [reply apps/backend/src/board/logic/place-path-card.test.ts:270]: LGTM

**Review by kayac-chang — APPROVED** [review]
LGTM

**Pull request merged**